app/i18n/locales/ja-JP.json

```diff
@@ -60,87 +66,90 @@
+  "wallet.restore.dialog.wallet.name.input.hint": "ウォレット名を入力してください",
   "wallet.restore.dialog.wallet.name.input.label": "ウォレット名",
-  "wallet.send.form.amount.equalsAda": "!!!equals {amount} ADA",
-  "wallet.send.form.amount.hint": "ドルでの金額",
+  "wallet.send.form.amount.equalsAda": "{amount}ADAと同額",
+  "wallet.send.form.amount.hint": "Adaでの金額",
   "wallet.send.form.amount.label": "金額",
   "wallet.send.form.description.hint": "任意のメッセージを入力できます",
   "wallet.send.form.description.label": "詳細",
   "wallet.send.form.errors.invalidAddress": "有効なアドレスを入力してください。",
   "wallet.send.form.errors.invalidAmount": "有効な金額を入力してください。",
-  "wallet.send.form.errors.invalidTitle": "３文字以上のタイトルを入力してください。",
+  "wallet.send.form.errors.invalidTitle": "３文字以上のタイトル名を入力してください。",
   "wallet.send.form.receiver.hint": "ウォレットアドレス",
   "wallet.send.form.receiver.label": "受取人",
   "wallet.send.form.submit": "送金",
   "wallet.send.form.title.hint": "例：太郎のための資金",
   "wallet.send.form.title.label": "タイトル",
-  "wallet.settings.assurance": "!!!Transaction assurance security level",
-  "wallet.settings.delete.dialog.confirmBackupNotice": "!!!Make sure you have access to backup before continuing. Otherwise, you will lose all your funds connected to this wallet.",
-  "wallet.settings.delete.dialog.confirmButtonLabel": "!!!Delete",
-  "wallet.settings.delete.dialog.enterRecoveryWordLabel": "!!!Enter the name of the wallet to confirm deletion:",
-  "wallet.settings.delete.dialog.title": "!!!Delete Wallet",
-  "wallet.settings.delete.dialog.wantToDeleteWalletQuestion": "!!!Do you really want to delete <strong>{walletName}</strong> wallet?",
-  "wallet.settings.deleteWalletButtonLabel": "!!!Delete wallet",
+  "wallet.settings.assurance": "トランズアクション保証のセキュリティレベル",
+  "wallet.settings.delete.dialog.confirmBackupNotice": "続行する前に、バックアップにアクセスできることを確認してください。さもなければこのウォレットに関連している全ての資産を失うことになります。",
+  "wallet.settings.delete.dialog.confirmButtonLabel": "削除",
+  "wallet.settings.delete.dialog.enterRecoveryWordLabel": "ウォレット名を入力して削除を確認してください:",
+  "wallet.settings.delete.dialog.title": "ウォレット削除",
+  "wallet.settings.delete.dialog.wantToDeleteWalletQuestion": "本当に<strong>{walletName}</strong>を削除しますか?",
+  "wallet.settings.deleteWalletButtonLabel": "ウォレット削除",
   "wallet.summary.no.transactions": "最近のトランズアクションはありません",
   "wallet.summary.page.pendingConfirmationLabel": "承認保留中",
   "wallet.summary.page.transactionsLabel": "トランズアクション数",
-  "wallet.transaction.assuranceLevel": "!!!Transaction assurance level",
-  "wallet.transaction.assuranceLevel.high": "!!!high",
-  "wallet.transaction.assuranceLevel.low": "!!!low",
-  "wallet.transaction.assuranceLevel.medium": "!!!medium",
-  "wallet.transaction.confirmations": "!!!confirmations",
-  "wallet.transaction.conversion.rate": "!!!Conversion rate",
-  "wallet.transaction.transactionId": "!!!Transaction ID",
+  "wallet.transaction.adaReceived": "!!!Ada received",
+  "wallet.transaction.adaSent": "!!!Ada sent",
```

Comment:
OK @darko-mijic. Will merge once I see these translations are added as the rest of this PR looks great ;)